### DISCUSSION TITLE
Trace csv export function

### DISCUSSION THREAD

**Discussion by maxzapp** [netbox-community/netbox#7596] [Q&A]
Hi,

i  am missing a way to export  all the traces in csv . The graphics is nice but at the end of the day i need a list of the full patching path  with more details like hight of each device. Number of  traces should be selectable by device, rack, room, etc.
Did i missed it only or is it not implemented?

Zappmax

**Comment by jeremystretch** [comment]
You can export cables directly from the UI. You also have the option of writing a [custom export template](https://netbox.readthedocs.io/en/stable/customization/export-templates/) if you'd like to include additional information.

**Reply by ziggekatten** [reply]
We faced the same situation and ended up with a custom script for this in the end to get a list of full cable traces in csv-format for a selected device interfaces and console ports. The script (if you select that option) delete the cable and create them again but connected to a dummy device in the racks. 

As there is no way of keeping a cable without a connection in netbox, this is how we keep track of reusable cables.

**Reply by maxzapp** [reply]
@jeremystretch i need to export the whole cable traces and it seems they can't be exported up til now. I don't have any knowledge in scooting right now.
@zi

**Reply by maxzapp** [reply]
@ziggekatten
Could you share the script pls but I don't understand why cables need to be deleted.

**Reply by maxzapp** [reply]
Hi ziggekatten

I am not a programmmer so could you please share your script, maybe i am able to adapt it to my needs

Many thx in advance
